src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -101,7 +101,8 @@ trait AttributeComponent {
     }
 
     def unmarshalAttributes(allAttributeRecsWithRef: Seq[(AttributeRecord, Option[EntityRecord])]): Map[String, Attribute] = {
-      allAttributeRecsWithRef.groupBy(_._1.name).map { case (name, attributeRecsWithRefForName) =>
+      allAttributeRecsWithRef.groupBy(_._1.name).map { case (name, attributeRecsWithRefForNameUndeduped) =>
```

Comment:
Un-de-duped? There were duplicates, you removed them, and then put them back?

"with dupes" might be better.